src/components/molecules/RadioField/RadioField.md

```diff
@@ -42,7 +42,7 @@ import { RadioField } from '@zopauk/react-components';
 ```jsx
 import { RadioField } from '@zopauk/react-components';
 
-<RadioField label="I'm checked by default" inputProps={{ value: 'radio4', name: 'radio4', defaultChecked: true }} />;
```

Comment:
I'm seeing in the console:
```Warning: Failed prop type: You provided a `checked` prop to a form field without an `onChange` handler. This will render a read-only field. If the field should be mutable use `defaultChecked`. Otherwise, set either `onChange` or `readOnly`.```

Reply:
🤔 I didn't think `defaultChecked` was a thing, I'll bring it back 👍🏻